### ISSUE TITLE
Cannot edit some fields

### ISSUE THREAD

**Issue by jhuddle61** [NEORIS-ZDEVOPS/DS-Editor#2]
Some of the fields in my dataset are shown in yellow and will not allow me to save changes to those fields. Is there something I can do to allow these fields to be editable?

![image](https://github.com/NEORIS-ZDEVOPS/DS-Editor/assets/26878954/ddf4ff11-7af0-4b70-81f1-801f21f8ae22)

**Comment by jhuddle61** [comment]
Found that fields defined as PIC 9 will not save edits. Is this by design?

**Comment by antonkc** [comment]
No, It should be editable. Today I am on vacation, I will try to recreate this problem tomorrow. Further information then.

**Comment by jhuddle61** [comment]
More info, I tried with a couple of values, one date and one integer both PIC 9, zeroes were saved for the value versus what I typed.

**Comment by antonkc** [comment]
It may be because the input is not validating. A better feedback system is in my backlog for these potential cases. Can you provide the full PIC and what numbers you tried to type in?

**Comment by antonkc** [comment]
Error reproduced

**Comment by antonkc** [comment]
Fix will roll in the next version. We plan to deploy a minor one this friday.

**Comment by jhuddle61** [comment]
Did the minor release get pushed last Friday?

**Comment by antonkc** [comment]
No, sorry. The code is ready, so I expect it to be published soon. The version number will 1.4.1 then.

PD: I didn't forget about this issue. You can expect an update as soon as the version rolls out.

**Comment by antonkc** [comment]
Some pipeline work must be done here, sorry. The one that uploads the artifact is a management person. I will try to get clearance to do it myself or with some script. Meanwhile, I am still stuck waiting with you.

I was too enthusiastic about the first issue opened by users that I jumped to giving dates without checking with said business person. Mistake on me.

Like a said before, I did not forget this issue and I will inform you as soon as it is realeased.
